plugins/toolkit/jetbrains-core/src/software/aws/toolkits/jetbrains/services/codemodernizer/ArtifactHandler.kt

```diff
@@ -111,9 +111,11 @@ class ArtifactHandler(private val project: Project, private val clientAdaptor: G
                     codeTransformRunTimeLatency = calculateTotalLatency(downloadStartTime, Instant.now()),
                     codeTransformJobId = job.id,
                     codeTransformTotalByteSize = totalDownloadBytes,
-                    codeTransformRuntimeError = telemetryErrorMessage
+                    codeTransformRuntimeError = telemetryErrorMessage,
```

Comment:
nit: extra comma

Reply:
Commas were added automatically when I formatted the file

Reply:
i'm fine with the trailing comma, we should enforce trailing as part of lint